README.md

```diff
@@ -48,6 +54,25 @@ docker run --rm -v "$PWD/tmp/000017_0.jsonl:/mydata/data.jsonl" \
            post -c ess /mydata/data.jsonl
 ```
 
+### queries v2
+
+https://docs.cyfronet.pl/display/FID/Queries+v2
+
+Either use directly the jsonl file or transform the original using (assuming the file is placed in `tmp/qv2-pub/000550_0`):
+```
+python transform/v2/tsv-to-jsonl.py tmp/qv2-pub/000550_0 > tmp/qv2-pub/000550_0.jsonl
+```
+
+To process datafiles without journal set envvar `OMIT_JOURNAL=1` for processing.
```

Comment:
Maybe we should create some dictionary of concepts. For example what it's the `journal`?

Reply:
Maybe, but maybe it should be kept on confluence? I don't know yet.